torch/_torch_docs.py

```diff
@@ -1305,6 +1313,17 @@
     -0.2239
     [torch.FloatTensor of size 2]
 
+    >>> x = torch.randn(2,5,4,2)
+    >>> torch.diagonal(x, -1, 1, 2)
```

Comment:
nit: Can you please add spaces between sizes in `randn`? Also, can you label the numbers with kwargs? The meaning of `-1, 1, 2` is not immediately obvious